File: src/1-Problem04.py
x = int(input("Masukan nilai X : "))
y = int(input("Masukan nilai Y : "))

# determine kuadran of x,y
if x == 0 and y == 0:
	daerah = "titik origin"
elif x == 0 :
	daerah = "sumbu Y"
elif y == 0 :
	daerah = "sumbu X"
elif x > 0 and y > 0 :
	daerah = "kuadran 1"
elif x < 0 and y > 0 :
	daerah = "kuadran 2"
elif x < 0 and y < 0 :
	daerah = "kuadran 3"
else :
	daerah = "kuadran 4"
	
print('Titik ({0},{1}) berada di {2}'.format(x, y, daerah))

File: src/1-Problem04.cpp
#include <iostream>


using namespace std;

int main()
{
    int X,Y;
    cout << "Masukkan nilai X : ";
    cin >> X;
    cout << "Masukkan nilai Y : ";
    cin >> Y;

    cout << "Titik (" << X << "," << Y << ") berada di ";

    if ((X == 0) && (Y == 0)) {
        cout << "titik origin" << endl;
    } else if (X == 0) {
        cout << "sumbu Y" << endl;
    } else if (Y == 0) {
        cout << "sumbu X" << endl;
    } else {
        cout << "kuadran ";
        if ((X > 0) && (Y > 0)) {
            cout << "1" << endl;
        } else if ((X < 0) && (Y > 0)) {
            cout << "2" << endl;
        } else if ((X < 0) && (Y < 0)) {
            cout << "3" << endl;
        } else if ((X > 0) && (Y < 0)) {
            cout << "4" << endl;
        }
    }

    return 0;
}
